### PR TITLE
fix(chat): fix file extension (Issue #1105)

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

fix file extension: back-end returns `application/octet-stream` for `GIF` or `PNG` entensions. Need to upload and save file extension in lowercase

Issues:

- Issue #1105 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1317.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/utils/app/file.ts [hunk 1]
```diff
@@ -361,3 +367,6 @@ export const getNextFileName = (
 
   return `${prefix}${maxNumber + 1}${defaultFileExtension}`;
 };
+
+export const prepareFileName = (filename: string) =>
```
Comment (Alexander-Kezik) [inline apps/chat/src/utils/app/file.ts:371]: Do we really need this? Why we can't just use filename?

Comment (IlyaBondar) [reply apps/chat/src/utils/app/file.ts:371]: @Alexander-Kezik , the reason is described in PR description

**Review by Alexander-Kezik — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1317.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1317.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**